deploy-jobs-template.yml

```diff
@@ -163,7 +163,32 @@ jobs:
                         "name": "WEBSITE_TIME_ZONE",
                         "value": "GMT Standard Time",
                         "slotSetting": false
-                      }
+                      },
+                      {
+                        "name": "AzureWebJobs.ExtractJobScheduler.Disabled",
+                        "value": "$(AzureWebJobsExtractJobSchedulerDisabled)",
+                        "slotSetting": true
+                      },
+                      {
+                        "name": "AzureWebJobs.ExtractJobNotificationTrigger",
```

Comment:
Missing .Disabled from end of this parameter.

I take this is a parameter from Library $(AzureWebJobsExtractJobSchedulerDisabled) - suggest this is not warranted, just set the values to true.  We never want to make these configurable via the library.

Reply:
Have corrected the var name. Library vars are not used just to configure the values, it's been done to keep the pipeline clean, tidy and lightweight.